src/components/Grants/Current/BudgetBanner.css

```diff
@@ -0,0 +1,39 @@
+.BudgetBanner {
+  box-sizing: border-box;
+  background: linear-gradient(89.29deg, #f6f5f8 0%, rgba(246, 245, 248, 0) 100%);
```

Comment:
Are we reusing this gradient? Maybe we can move it to the theme

Reply:
Good idea, it is the same as the [RequestBanner.css](https://github.com/decentraland/governance/pull/692/files#diff-70ed416cd8c009a1f0f435613fbfc8a6c7f5960805946b6e60e55529c4dd3900)